src/main/java/com/cburch/logisim/circuit/Circuit.java

```diff
@@ -987,4 +987,14 @@ public void setDownloadFrequency(double value) {
     staticAttrs.setValue(CircuitAttributes.DOWNLOAD_FREQUENCY, value);
     if (proj != null) proj.setForcedDirty();
   }
+
+  public String getDownloadBoard() {
+    return staticAttrs.getValue(CircuitAttributes.DOWNLOAD_BOARD);
+  }
+
+  public void setDownloadBoard(String board) {
+    if (board.equals(staticAttrs.getValue(CircuitAttributes.DOWNLOAD_BOARD))) return;
```

Comment:
I think negating the condition, removing `return` and keeping the remaining code conditional would simplify this block.

Reply:
done